src/fem/dh_cell_accessor.hh

```diff
@@ -53,7 +53,7 @@ public:
 
     /// Return local index to element (index of DOF handler).
     inline unsigned int local_idx() const {
-    	ASSERT_LT_DBG(loc_ele_idx_, dof_handler_->el_ds_->lsize()+dof_handler_->ghost_4_loc.size()).error("Local element index is out of range!\n");
+        ASSERT_LT_DBG(loc_ele_idx_, dof_handler_->global_to_local_el_idx_.size()).error("Local element index is out of range!\n");
```

Comment:
There should be local_to_global_el_idx, isn't it?

Reply:
There is no local_to_global_element map. We need just the number of elements on which the DH has information about dofs.